### PR TITLE
Add premium warning

### PR DESCRIPTION
**Pull request by RobertCallens**
add the message "This is not a recurring subscription. We won't automatically charge you after your premium time runs out!" to the premium dashboard #715

### CODE REVIEW

**Comment by RobertCallens** [comment]
changed volume to supply on the features tab #705

**Comment by RobertCallens** [comment]
fuffiled #703

File: components/Flipper/Flip/Flip.tsx [hunk 1]
```diff
@@ -147,7 +147,7 @@ function Flip(props: Props) {
                             <b style={{ color: 'red' }}>{formatPrices(props.flip.cost)} Coins</b>
                         </p>
                     )}
-                    {settings.hideMedianPrice ? null : (
+                    {settings.hideTargetPrice ? null : (
                         <p>
                             <span>Median price: </span>
```
Comment (Ekwav) [inline components/Flipper/Flip/Flip.tsx:152]: this was forgotten

**Review by Ekwav — CHANGES_REQUESTED** [review]

File: components/Flipper/Flip/Flip.tsx (hunk 1 above)
Comment (matthias-luger) [reply components/Flipper/Flip/Flip.tsx:152]: I thought we agreed, that the name "hideMedianPrice" would stay?

Comment (Ekwav) [reply components/Flipper/Flip/Flip.tsx:152]: I still don't care about the property name, the display name is not updated

File: api/ApiHelper.tsx [hunk 2]
```diff
@@ -635,7 +635,7 @@ export function initAPI(returnSSRResponse: boolean = false): API {
                 estProfit: !flipSettings.hideEstimatedProfit,
                 lbin: !flipSettings.hideLowestBin,
                 slbin: !flipSettings.hideSecondLowestBin,
-                medPrice: !flipSettings.hideMedianPrice,
+                medPrice: !flipSettings.hideTargetPrice,
```
Comment (matthias-luger) [inline api/ApiHelper.tsx:638]: Please dont rename the used property. Revert all name changes

File: components/Premium/Premium.tsx [hunk 3]
```diff
@@ -122,6 +122,8 @@ function Premium() {
                     >
                         <span>Your premium ends: {moment(hasPremiumUntil).fromNow()}</span>
                     </OverlayTrigger>
+
+                    <p style={{ color: 'yellow' , marginTop: 15 }}>This is not a recurring subscription. We won't automatically charge you after your premium time runs out!</p>
```
Comment (matthias-luger) [inline components/Premium/Premium.tsx:126]: Put the text under the "Premium Plans" headline

**Review by matthias-luger — CHANGES_REQUESTED** [review]

File: components/Premium/Premium.tsx (hunk 3 above)
Comment (matthias-luger) [reply components/Premium/Premium.tsx:126]: As there is no "Premium Plans" anymore, I would put this information as a sub-title under the 
"Buy premium for a certain duration with your CoflCoins. The premium activate shortly after your purchase."-Title

**Comment by matthias-luger** [comment]
@ByteGamer 
Do you plan on taking a look at the issues? 
No problem if you dont have time to do so, just so I know that I can do them and you dont have uncommited changes :)

**Comment by RobertCallens** [comment]
i mean i dont think that this is nessesary with the new payment system

**Pull request closed without merging**